### PR TITLE
Adding Monero

### PR DESCRIPTION
**Pull request by serhack**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:

  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)

- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:
Monero is a cryptocurrency focused on privacy. A topic page should be curated.
At this moment, It has a volume of 43,033,600 dollars. Privacy is enabled by default that means nobody can see your informations. 

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Hi @serhack! Are you affiliated in any way with the Monero project?

**Comment by serhack** [comment]
What do you mean by affiliated?

**Comment by MikeMcQuaid** [comment]
@serhack Are you either a maintainer of the project or have any paid relationship with Monero?

**Comment by serhack** [comment]
Monero is a community-funded project, I can confirm that I have a paid relationship with Monero.

**Comment by MikeMcQuaid** [comment]
Ok, thanks!

File: topics/monero/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+aliases: monero-cryptocurrency, monero-crypto
+created_by: Monero
+display_name: Monero
+github_url: https://github.com/monero-project
+released: April 18th, 2014
+short_description: Monero is a private, fungible, open source, decentralized cryptocurrency.
+topic: monero
+url: https://getmonero.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Monero_(cryptocurrency)
+---
```
Comment (cheshire137) [inline topics/monero/index.md]: Can you add a full description after this line? Just something that expands on what the "short_description" says. Maybe compare Monero to other cryptocurrencies.

Comment (serhack) [reply topics/monero/index.md]: Okay, I will. Thanks

Comment (serhack) [reply topics/monero/index.md]: Done :)

**Comment by serhack** [comment]
Any news? :)

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to github/explore, @serhack! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by serhack** [comment]
Yeah, thanks @mikemcquaid !

**Pull request merged**